src/NodaTime/Instant.cs

```diff
@@ -502,6 +502,13 @@ public static Instant Max(Instant x, Instant y)
         #endregion
 
         /// <summary>
+        /// Constructs a <see cref="double"/> representing the same instant of time as this value.
+        /// </summary>
+        /// <returns>Returns a <see cref="double"/> containing the number of whole and fractional days since the Julian Epoch (Noon January 1st, 4713 BCE)</returns>
+        [Pure]
+        public double ToJulianDayNumber() => this.duration.TotalSeconds / 86400d + 2440587.5;
```

Comment:
That would work, I used TotalSeconds since I was referencing the UnixEpoch in the other function and wanted to keep them consistent.